src/librustc_resolve/build_reduced_graph.rs

```diff
@@ -390,7 +388,7 @@ impl<'b> Resolver<'b> {
                 debug!("(building reduced graph for external crate) building module {} {:?}",
                        name, vis);
                 let parent_link = ModuleParentLink(parent, name);
-                let module = self.new_module(parent_link, Some(def), true);
+                let module = self.new_module(parent_link, Some(def), DUMMY_NODE_ID);
```

Comment:
Will do, but that would still be enforced dynamically (with `.unwrap()`) unless we separate local and external modules at the type level.